Taskfile.yml

```diff
@@ -0,0 +1,34 @@
+#
+# Copyright (C) 2015 The Gravitee team (http://gravitee.io)
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#         http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+version: '3'
+
+env:
+  DOCKER_TAG: local
+
+tasks:
+  build-quick:
+    desc: "Build management-api & gateway quickly without running tests & validations steps"
+    cmds:
+      - mvn clean install -DskipTests -Dskip.validation -P gio-dev -T 1C
+
+  docker:
+    desc: "Build all 🐳 images"
+    cmds:
+      - docker build -t graviteeio.azurecr.io/apim-gateway:$DOCKER_TAG -f gravitee-apim-gateway/docker/Dockerfile ./gravitee-apim-gateway/gravitee-apim-gateway-standalone/gravitee-apim-gateway-standalone-distribution/target
+      - docker build -t graviteeio.azurecr.io/apim-management-api:$DOCKER_TAG -f gravitee-apim-rest-api/docker/Dockerfile ./gravitee-apim-rest-api/gravitee-apim-rest-api-standalone/gravitee-apim-rest-api-standalone-distribution/target
+      - docker build -t graviteeio.azurecr.io/apim-management-ui:$DOCKER_TAG -f gravitee-apim-console-webui/docker/Dockerfile ./gravitee-apim-console-webui
+      - docker build -t graviteeio.azurecr.io/apim-portal-ui:$DOCKER_TAG -f gravitee-apim-portal-webui/docker/Dockerfile ./gravitee-apim-portal-webui
```

Comment:
thought (non-blocking): It won't work if build has not been done prior to that task but I'm not sure if we should use `deps` field here.

Reply:
Maybe we can do that later. Currently, there is no tasks to build the UIs. They have to be built "manually".

Reply:
Yes, sure. It was just a thought. ;)